src/components/viewer/viewer-elements/AuthorActionControls.tsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import classNames from 'classnames';
 import { connect } from 'react-redux';
-import styles from './Viewer.scss';
+import styles from '../Viewer.scss';
+import { Icon } from 'react-icons-kit';
+import { ic_mode_edit } from 'react-icons-kit/md/ic_mode_edit';
+import { ic_delete } from 'react-icons-kit/md/ic_delete';
 import {
   hideViewer,
   openViewerDeleteModal,
   showEditorAnnotation,
 } from 'store/widgets/actions';
 import { AnnotationAPIModel } from 'api/annotations';
 import Timer = NodeJS.Timer;
-import ViewerItemDialog from './ViewerItemDialog';
-import ppGA from '../../pp-ga';
+import ppGA from '../../../pp-ga';
 
-interface IViewerItemControlsProps {
+interface IAuthorActionControlsProps {
```

Comment:
A nice decision to introduce `AuthorAction` and `UserActionControls` -- very semantic for the developer.